src/App.tsx

```diff
@@ -48,99 +51,65 @@ const App: React.FunctionComponent = () => {
       case 'secondInput': { setSecond(target.value); break }
     }
   }
-  const handleSubmit = (e: React.FormEvent<HTMLInputElement>): void => {
-    e.preventDefault()
+  const handleSubmit = (): any => {
```

Comment:
Type the same way to find a better option for `any` here.

Reply:
Thank you for your share. 'setInterval()' should return an ID which is number type. The 'handleSubmit' function doesn't return any value.
So, when I switch the funtion type to other a specific type, comes up a problem.